mslib/mswms/wms.py

```diff
@@ -146,7 +146,7 @@ def verify_pw(username, password):
 from mslib.utils.coordinate import get_projection_params
 
 # Logging the Standard Output, which will be added to the Apache Log Files
-logging.basicConfig(level=logging.DEBUG,
```

Comment:
keep in the whole MSS code the existing DEBUG. 

On the server site a user has to use their own plot class, data classes to extend the existing. Usually that did not work on first attempt. They need these debug messages.